faker/providers/phone_number/en_US/__init__.py

```diff
@@ -4,28 +4,34 @@
 
 class Provider(PhoneNumberProvider):
     formats = (
-        '+##(#)##########',
-        '+##(#)##########',
-        '0##########',
-        '0##########',
+        # Standard 10-digit phone number formats
+        '##########',
+        '##########',
         '###-###-####',
-        '(###)###-####',
-        '1-###-###-####',
-        '###.###.####',
         '###-###-####',
+        # Optional 10-digit local phone number format
         '(###)###-####',
-        '1-###-###-####',
+        '(###)###-####',
+        # Non-standard 10-digit phone number format
+        '###.###.####',
         '###.###.####',
+        # Standard 11-digit phone number format
+        '1-###-###-####',
+        '1-###-###-####',
+        # Standard 10-digit phone number format with extensions
         '###-###-####x###',
-        '(###)###-####x###',
-        '1-###-###-####x###',
-        '###.###.####x###',
         '###-###-####x####',
-        '(###)###-####x####',
-        '1-###-###-####x####',
-        '###.###.####x####',
         '###-###-####x#####',
-        '(###)###-####x#####',
+        # Optional 10-digit local phone number format with extensions
+        '(###)###-###x###',
+        '(###)###-###x####',
+        '(###)###-###x#####',
```

Comment:
These 9-digit phone numbers won't be valid for the US. I'm sure it's just a typo, but a valid US phone needs 4 digits at the end before the extension.

Reply:
Ya, I haven't had time to come through and fix all of these yet, it will probably be a while so this will likely sit lazy for a bit.